eng/scripts/job-matrix/job-matrix-functions.modification.tests.ps1

```diff
@@ -119,6 +119,46 @@ Describe "Platform Matrix Import" -Tag "import" {
         $matrix[4].parameters.Foo | Should -Be "foo2"
     }
 
+    It "Should generate different matrices when importing and not importing" {
```

Comment:
I was hoping to see another matrix entry here that is equal to the one doing the import so that we I could easily see what the full matrix would look like.

Reply:
```
{
  "matrix": {
    "Foo_Bar" [ ... ]
    "Foo_Bar_Baz" [ ...]
    "testField1": [ "test11", "test12" ]
  },
}
```

Something like that, or what it actually represents.